metadata.yaml

```diff
@@ -24,6 +24,26 @@ missions:
   description: The Health Check mission demonstrates the health check pattern. When you deploy an application, its important to know if it is available and if it can start handling incoming requests. Implementing the health check pattern allows you to monitor the health of an application, which includes if an application is available and whether it is able to service requests.
   metadata:
     level: foundational
+- id: istio-circuit-breaker
```

Comment:
As we will use the F8 Launcher filter option, then this field should be added for every istio mission under `metadata` -> https://github.com/snowdrop/launcher-booster-catalog/blob/istio/metadata.yaml#L44
@kenfinnigan Can you take care of that ?